### PR TITLE
modules: add chat

### PR DESCRIPTION
**Pull request by rmackay9**
This new page gives instructions on how prerequisites for running the new chat module (see https://github.com/ArduPilot/MAVProxy/pull/1278)

I've tested this locally and it looks OK.

### CODE REVIEW

File: mavproxy/source/docs/modules/chat.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+The Chat module requires an OpenAI API key.  Details on how to get an API key are on the `OpenAI Account setup page  <https://platform.openai.com/docs/quickstart/account-setup>`__.  ArduPilot core developers can request a key in the "dev-team-funding" Discord channel.
+
+If an ArduPilot provided API Key is not used then the OpenAI account holder will need to re-create the "ArduPilot Vehicle Control via MAVLink" assistant which can be done by running the MAVProxy/modules/mavproxy_chat/assistant_setup/setup_assistant.py script.
+
+The pyaudio, wave and openai python packages must be installed manually.
+
+Windows
+-------
+
+Open a command prompt and enter the following commands:
```
Comment (Hwurzburg) [inline mavproxy/source/docs/modules/chat.rst:29]: as admin or normal user?

Comment (rmackay9) [reply mavproxy/source/docs/modules/chat.rst:29]: Txs for the review.  ON windows, using just the normal users seems fine.

Comment (Hwurzburg) [reply mavproxy/source/docs/modules/chat.rst:29]: should we hold off on this until you get the Windows issue sorted?requested a key to test the procedure out

Comment (rmackay9) [reply mavproxy/source/docs/modules/chat.rst:29]: Hi @Hwurzburg, I figured out the issue today actually so it's working on Windows as well and was merged at the EU Dev call earlier..

**Comment by rmackay9** [comment]
Thanks so much!

**Pull request merged**